Source/SpiderEye.Linux/GtkWebview.cs

```diff
@@ -260,21 +289,20 @@ private async void UriSchemeCallback(IntPtr request, IntPtr userdata)
                     }
                 }
 
-                FinishUriSchemeCallbackWithError(request);
+                FinishUriSchemeCallbackWithError(request, 4); // 4 = file not found
```

Comment:
Couldn't you create constants for these magic numbers?

Reply:
Ups, fixed